packages/ag-charts-community/src/scene/scene.ts

```diff
@@ -59,18 +47,9 @@ export class Scene {
         return this.pendingSize?.[1] ?? this.canvas.height;
     }
 
-    setContainer(value: HTMLElement | DOMManagerLike) {
-        const isElement = (v: unknown): v is HTMLElement => {
-            return typeof (v as any).tagName !== 'undefined';
-        };
-        if (isElement(value)) {
-            const { element } = this.canvas;
-            element.parentElement?.removeChild(element);
-            value.appendChild(element);
-        } else {
-            this.domManager = value;
-            this.domManager.addChild('canvas', 'scene-canvas', this.canvas.element);
-        }
+    setContainer(container: HTMLElement) {
+        this.canvas.element.remove();
+        container.appendChild(this.canvas.element);
```

Comment:
I'd prefer if `DOMManager` was still responsible for creation + destruction of the canvas element.

Reply:
Do we still have a use-case for `Scene.setContainer` (e.g. integrated charts)?

The `Scene.domManager` is only used in the constructor, so the coordination between the constructor and `setContainer` doesn't make a lot of sense. It would be cleaner if the `Scene` constructor just took a non-nullable and `HTMLCanvasElement`, and leave it up to the caller to add that element in the correct place.

Reply:
So yes the Integrated Charts case is real and exists - without this change, switching chart types yield a janky experience because we remove the old Chart instance and it's Canvas, then there maybe several frames before a new canvas is put back in place and the Chart rendered.

Reply:
Done. Change reverted.